docs/quickstart.rst

```diff
@@ -34,20 +35,40 @@ On Ubuntu 14.04 and 16.04, this works:
 
 .. code-block:: bash
 
-    sudo apt-get update
+    $ sudo apt-get update
 
-    sudo apt-get install libffi-dev
+    $ sudo apt-get install libffi-dev
 
 On Fedora 23 and 24, this works:
 
 .. code-block:: bash
 
-    sudo dnf update
+    $ sudo dnf update
 
-    sudo dnf install libffi-devel
+    $ sudo dnf install libffi-devel
 
 For other operating systems, just do some web searches for "ffi.h" with the name of your OS.
 
+Dependency 2: libssl-dev
+^^^^^^^^^^^^^^^^^^^^^^^^
+BigchainDB (server and driver) also depends on `cryptography`_,
+which in turn depends on `libssl`_ AND `libcrypto`_.
+Hence, depending on your setup you need to install the `libssl-dev`_ (Ubuntu)
+OR `openssl-devel`_ (RHEL) package, which installs the development
+libraries and header files for `libssl`_ and `libcrypto`_.
+
+On Ubuntu 14.04 and 16.04:
+
+.. code-block:: bash
+
+    $ sudo apt-get install libssl-dev
+
+On Fedora 23 and 24:
+
+.. code-block:: bash
+
+    sudo dnf install openssl-devel
+
```

Comment:
Below, Python 3.5+ now becomes "Dependency 3". The following ones have to be renumbered as well.